### PR TITLE
Add label to ts image

### PR DESCRIPTION
**Pull request by chen3933**
*GitHub Issue #, if available:*

**Note**: 
- If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

- All PR's are checked weekly for staleness. This PR will be closed if not updated in 30 days.

### Description

reference PR : https://github.com/aws/deep-learning-containers/pull/2647

### Tests run

**NOTE: By default, docker builds are disabled. In order to build your container, please update dlc_developer_config.toml and specify the framework to build in "build_frameworks"**

- [x] x86 pt 2.0 test [654d0c5](https://github.com/aws/deep-learning-containers/pull/3155/commits/654d0c5b268d80fe1825b1259ef506909e2063ed)

- [x] graviton pt 2.0 test [826b930](https://github.com/aws/deep-learning-containers/pull/3155/commits/826b930b50f454411f7ac73ae9589b427aaf1691)

- [x] x86 pt 1.13 test [8e48093](https://github.com/aws/deep-learning-containers/pull/3155/commits/8e48093e606579c16a9b5831c41693e9f2bf8390)

- [x] neuron pt 1.13 test [b476cc7](https://github.com/aws/deep-learning-containers/pull/3155/commits/b476cc72dc15bb816f3a4292c80e3be84fbbd614)

- [x] neuronx pt 1.13 test [0b79a4a](https://github.com/aws/deep-learning-containers/pull/3155/commits/0b79a4a74a49f58323082bb1583419dade0fc17e)

### Formatting
- [x] I have run `black -l 100` on my code (formatting tool: https://black.readthedocs.io/en/stable/getting_started.html)

### Additional context

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pytorch/inference/docker/1.11/py3/Dockerfile.cpu [hunk 1]
```diff
@@ -2,6 +2,10 @@ ARG PYTHON=python3
 ARG PYTHON_VERSION=3.8.13
```
Comment (arjkesh) [inline pytorch/inference/docker/1.11/py3/Dockerfile.cpu:2]: Can we revert the changes for PT 1.11 and 1.12? These images are past due as per our support policy https://aws.amazon.com/releasenotes/dlc-support-policy/

File: test/dlc_tests/sanity/test_pre_release.py [hunk 2]
```diff
@@ -234,6 +235,24 @@ def test_tf_serving_api_version(tensorflow_inference):
         stop_and_remove_container(container_name, ctx)
 
 
+@pytest.mark.usefixtures("sagemaker_only")
```
Comment (arjkesh) [inline test/dlc_tests/sanity/test_pre_release.py:238]: thank you for adding these

**Review by arjkesh — COMMENTED** [review]
In order to maintain consistency (versions that use dashes instead of "." as separators), we may want to move these versions into the buildspec

File: pytorch/inference/docker/1.11/py3/Dockerfile.cpu (hunk 1 above)
Comment (chen3933) [reply pytorch/inference/docker/1.11/py3/Dockerfile.cpu]: Done

**Comment by arjkesh** [comment]
These changes look good, thank you - @catalin-manciu-aws would you be able to review the neuron changes?

**Review by catalin-manciu-aws — APPROVED** [review]

File: pytorch/inference/docker/1.13/py3/cu117/Dockerfile.gpu [hunk 3]
```diff
@@ -12,7 +12,7 @@ ARG TORCH_URL=https://aws-pytorch-unified-cicd-binaries.s3.us-west-2.amazonaws.c
 ARG TORCHVISION_VERSION=0.14.1+cu117
 ARG TORCHAUDIO_VERSION=0.13.1+cu117
 ARG TORCHDATA_VERSION=0.5.1
-ARG TORCHSERVE_VERSION=0.7.1
+ARG TORCHSERVE_VERSION
 ARG SM_TOOLKIT_VERSION=2.0.14
```
Comment (arjkesh) [inline pytorch/inference/docker/1.13/py3/cu117/Dockerfile.gpu:16]: toolkit version can be removed here, right?

Comment (chen3933) [reply pytorch/inference/docker/1.13/py3/cu117/Dockerfile.gpu]: Right. removed.

**Comment by chen3933** [comment]
@catalin-manciu-aws would you please review again. I revert the toml to pass the build.

**Review by catalin-manciu-aws — APPROVED** [review]
